### PR TITLE
add option --output-for-linaro-automation to compare.py call

### PR DESCRIPTION
**Pull request by jduraj**

### CODE REVIEW

File: benchmarks/comparison.py [hunk 1]
```diff
@@ -9,14 +9,14 @@
 from django.conf import settings
 
 compare_script = os.getenv('COMPARE_SCRIPT', None)
-compare_command = [compare_script, testjob_before.data.file.name, testjob_after.data.file.name]
+compare_command = [compare_script]
 if not compare_script:
```
Comment (mwasilew) [inline benchmarks/comparison.py]: This is always False. Even if compare_script == None than compare_command = [None]. This always resolves as logical True, so negation  will always be False. The if statement is either not needed here or shadows some bug.

Comment (jduraj) [inline benchmarks/comparison.py]: I have to admit that it is not clear. The comparison is done with compare_script, not compare_command. I can refactor like this if it makes more sense:
`compare_script = os.getenv('COMPARE_SCRIPT', None)
if not compare_script:
    compare_script = os.path.join(settings.EXTERNAL_DIR['BASE'], 'art-testing', 'compare.py')
    compare_command = [compare_script, '--output-for-linaro-automation']
else:
    compare_command = [compare_script]
`

**Comment by mwasilew** [comment]
Julien, sorry my bad. Your previous code was better (without 'else'). Could you do the following:
 - squash this into single commit
 - keep the lines no more than 80 chars long (thus update line 13 which is longer)
?

**Comment by jduraj** [comment]
I reverted to the code without 'else', then also fixed a couple of other lines that were 80+ chars long, and squashed it into one commit.

**Comment by mwasilew** [comment]
Looks good. Thanks Julien!

**Pull request merged**